src/Client.php

```diff
@@ -25,6 +25,9 @@ public function __construct(
         }
         $stack = $config['handler'];
 
+        // Download file without decoding
+        $config['decode_content'] = false;
```

Comment:
Nasiel som issue na Guzzle: https://github.com/guzzle/guzzle/issues/2146#issuecomment-424515055
Neviem ci to suvisi s poslednym releasom, .... v kazdom pripade HTTP extractor ma iba stahovat CSV subory, a nema robit ziadne dekodovanie. tak to tu vypinam.